features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -355,6 +382,21 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         }
     };
 
+    const hybridFlowConfigValuesChangeListener = (tempForm: Map<string, FormValue>): void => {
+
+        const withPredicate = (val: string): boolean => val === HYBRID_FLOW_ENABLE_CONFIG ;
```

Comment:
`withPredicate` is too complex term, IMO. Shall we directly pass this filter function to `.find()` method instead of defining another constant?

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/6271/commits/11295ba4d8273091b1934290f09ab0ab4d02ac0d